src/rp2040/rp2040_link.lds.S

```diff
@@ -20,20 +20,21 @@ SECTIONS
     .text : {
         . = ALIGN(4);
         KEEP(*(.boot2))
-        _text_vectortable_start = .;
-        KEEP(*(.vector_table))
-        _text_vectortable_end = .;
-        *(.text .text.*)
-        *(.rodata .rodata*)
+        KEEP(*(.vector_table_flash))
```

Comment:
The only VectorTableFlash , ResetHandler and reset_handler_stage_two are executed from flash. reset_handler_stage_two moves everything to the ram.